### PR TITLE
[spec] Updates to spec commands and documentation

### LINKED ISSUE #2165: Document how to run individual mocha tests in CONTRIBUTING.md
Document how to run individual mocha tests in `CONTRIBUTING.md` so people can run individual aXe tests on specific components.

### PR DESCRIPTION
**Pull request by el-mapache**
## Description

* Adds more fine grained controls to run individuals spec files
* Adds commands to run the visual tests and golden master generation
* Changes the name of the chrom-headless spec file to prevent it from
auto-running

Fixes #2165

### CODE REVIEW

**Comment by maya** [comment]
Looking good. Do you have an example of this: 
>Alternatively, you can add an `.only` to a `describe` or `it` block, and run the `npm run test` command.

**Comment by maya** [comment]
For `npm run mocha ${path/to/spec-file}` is it like `npm run mocha spec/screenshots/accordion_large-desktop.png` will run visual regression tests? Are there other files in spec and other tests this will run?

**Comment by el-mapache** [comment]
`mocha` will only run javascript files, passing an image to it should result in an error. I specifically called out `spec-file` in the example since all of our tests have `.spec.js`. I can add that explicitly in the readme

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -88,6 +88,15 @@ bin)/gulp` instead of `gulp`.)
 Note that running the tests also requires an installation of
 Chrome v59 or higher (v60 if you're on Windows).
 
+If you want to run a single test file, run `npm run mocha ${path/to/spec-file}`,
+substituting the actual path to the spec. Only javascript files can be executed by the `mocha` runner,
+and only those js files in the `spec` directory ending with a `.spec.js`.
+
+Alternatively, you can add an `.only` to a `describe` or `it` block (i.e. describe.only('my spec'))
```
Comment (maya) [inline CONTRIBUTING.md]: @el-mapache Should describe.only('my spec')) be in code ticks ie `describe.only('my spec'))`?

**Pull request merged**